pkgs/development/octave-modules/symbolic/default.nix

```diff
@@ -0,0 +1,44 @@
+{ buildOctavePackage
+, lib
+, fetchurl
+, python
+, python2Packages
```

Comment:
avoid passing in both the interpreter and the package set. Choose one. Both contain each other.

Reply:
I actually have to do that. The python version that gets passed in is from Octave's default.nix, which is python3. But, symbolic is built assuming an older version of sympy, which is only contained in python2's packages. However, symbolic requires python3 be present.